srcpkgs/firefox-esr/files/mozconfig

```diff
@@ -32,4 +32,5 @@ ac_add_options --enable-optimize="$CFLAGS"
 
 ac_add_options --enable-official-branding
 ac_add_options --enable-application=browser
+
```

Comment:
why?

Reply:
I sync the files from src pkg/firefox so it is easier to diff them in the future.